chrome-extensions/CapSolver/background.js

```diff
@@ -468,7 +468,7 @@
             enabledForBlacklistControl: !1,
             blackUrlList: [],
             isInBlackList: !1,
-            reCaptchaMode: "token",
```

Comment:
do we really have to change the option in background.js as well? It makes sense to have it synced but just wondering if it's enough to do it in config.js only as this is the configuration part that can be modified via extension UI?

Reply:
if this step is necessary it adds additional thing to take care about when updating Capsolver version, since background.js already requires adding keepAlive at the end so prettify creates ton of differences so it will be easy to forget and override settings from config.

Reply:
oops... I tested it again and actually no. To be honest, it wouldn't do any harm, but we can rule that out (so it doesn't end up on the update list next time). Thanks for mentioning this 😃

Reply:
nice, that simplifies things 😃